lib/util/args.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Burrito.Util.Args do
+  @moduledoc """
+  This module provides a method to help fetch CLI arguments passed down
+  from the Zig wrapper binary. To get around the many issues of escaping quotes
+  in Windows, we base64 encode each argument before passing it to the Erlang VM.
+  `get_arguments/0` will decode them automatically for you.
+  """
+
+  @spec get_arguments :: list(String.t())
+  def get_arguments do
+    if is_windows?() && args_are_encoded?() do
+      {prefix, trimmed} = :init.get_plain_arguments() |> Enum.split(4)
```

Comment:
I'm a little bit wary of using `4` as a magic number here for the split, since it assumes that the application is being run with `--no-halt +iex -S mix`. Is there a more portable approach we could use? Maybe putting a sentinel value in here that we can use to partition the list of arguments?

Reply:
yeah I was stuck on this as well - my fear is introducing a terminal value that clashes with something someone wants to pass in, although I suppose we could split on something like the first `@` we find?

Reply:
Yeah, I've got similar concerns. I'm thinking that we merge as-in to unblock Windows users with the code that you've been testing, but we can make an issue to track the possible issue and think about a better solution.